frontend/pages/index.page.tsx

```diff
@@ -191,53 +172,35 @@ const FeaturedContent: FC = () => {
   }, [loading]);
 
   return (
-    <>
-      <Box
-        sx={{
-          flex: "1 1",
-          display: "flex",
-          alignItems: "center",
-          justifyContent: "center",
-          margin: "1.5rem 1rem 1.5rem 1rem",
-          p: 4,
-        }}
-      >
-        <Card elevation={5}>
-          <Box sx={{ m: 3, alignItems: "center", justifyContent: "center" }}>
-            <Grid container alignItems="center" justifyContent="center">
-              <Grid item>
-                <Typography variant="h6" gutterBottom component="div" fontWeight="bold">
-                  Featured Content
-                </Typography>
+    <Box>
+      <Box sx={{ m: 3, alignItems: "center", justifyContent: "center" }}>
+        <Grid container alignItems="center" justifyContent="center">
+          <>
+            {items.length ? (
+              items.map((module, index) => (
+                <Grid item key={index} xs={12} sm={6} md={4} lg={3} xl={3}>
+                  <Link href={module.absoluteUrl} underline="none">
+                    <a>
+                      <ModuleUnionCard module={module} />
+                    </a>
+                  </Link>
+                </Grid>
+              ))
+            ) : loading ? (
+              <Grid item xs={12} sm={6} md={4} lg={3} xl={3}>
+                <StyledCard>
+                  <CardContent>
+                    <Skeleton sx={{ minHeight: 200, width: "100%" }} />
+                  </CardContent>
+                </StyledCard>
               </Grid>
-            </Grid>
-            <Grid container alignItems="center" justifyContent="center">
-              <>
-                {items.length ? (
-                  items.map((module, index) => (
-                    <Link href={module.absoluteUrl} underline="none" key={index}>
-                      <a>
-                        <Grid item>
-                          <ModuleUnionCard module={module} key={index} />
-                        </Grid>
-                      </a>
-                    </Link>
-                  ))
-                ) : loading ? (
-                  <Card>
-                    <CardContent>
-                      <Skeleton sx={{ minHeight: 200, width: "100vw" }} />
-                    </CardContent>
-                  </Card>
-                ) : (
-                  <p>Sorry, there are no featured contents to display. Please check back later!</p>
-                )}
-              </>
-            </Grid>
-          </Box>
-        </Card>
+            ) : (
+              <p>Sorry, there are no featured contents to display. Please check back later!</p>
+            )}
```

Comment:
Wasn't introduced in this PR, but can we change this to "There is no featured content ..."?